src/V3Case.cpp

```diff
@@ -210,26 +222,36 @@ class CaseVisitor final : public VNVisitor {
                     const uint32_t val = numval.toUInt();
 
                     uint32_t firstOverlap = 0;
-                    bool foundOverlap = false;
+                    AstNode* overlappedCondp = nullptr;
                     bool foundHit = false;
                     for (uint32_t i = 0; i < numCases; ++i) {
                         if ((i & mask) == val) {
                             if (!m_valueItem[i]) {
-                                m_valueItem[i] = itemp;
+                                m_valueItem[i] = icondp;
                                 foundHit = true;
-                            } else if (!foundOverlap) {
+                            } else if (!overlappedCondp) {
                                 firstOverlap = i;
-                                foundOverlap = true;
+                                overlappedCondp = m_valueItem[i];
                                 m_caseNoOverlapsAllCovered = false;
                             }
                         }
                     }
                     if (!nodep->priorityPragma()) {
                         // If this case statement doesn't have the priority
                         // keyword, we want to warn on any overlap.
-                        if (!reportedOverlap && foundOverlap) {
-                            icondp->v3warn(CASEOVERLAP, "Case values overlap (example pattern 0x"
-                                                            << std::hex << firstOverlap << ")");
+                        if (!reportedOverlap && overlappedCondp) {
+                            std::ostringstream examplePattern;
+                            if (isCondWildcard) {
```

Comment:
I'm ok with this, or alternatively always print the example pattern even if not wildcarded.

Reply:
Since the overlapping item has been pointed out specifically (like [here](https://github.com/verilator/verilator/blob/ec2e3ec0e4f398795b3a2f9046723384a5c95f2d/test_regress/t/t_param_scope_bad.out#L2)), I think add another 'example pattern' in such condition is redundant, so only print the message as wildcard condition's special case.